internal/dao/helm_history.go

```diff
@@ -55,12 +56,23 @@ func (h *HelmHistory) List(ctx context.Context, _ string) ([]runtime.Object, err
 
 // Get returns a resource.
 func (h *HelmHistory) Get(_ context.Context, path string) (runtime.Object, error) {
-	ns, n := client.Namespaced(path)
+	fqn, rev, _ := strings.Cut(path, ":")
```

Comment:
I think we should check if we the rev is set here. Otherwise all bets are off.

Reply:
`strings.Cut` can safely handle the case where there is no rev (e.g. `path = "default/my-redis"`), here is the official implementation:
```go
// Cut slices s around the first instance of sep,
// returning the text before and after sep.
// The found result reports whether sep appears in s.
// If sep does not appear in s, cut returns s, "", false.
func Cut(s, sep string) (before, after string, found bool) {
	if i := Index(s, sep); i >= 0 {
		return s[:i], s[i+len(sep):], true
	}
	return s, "", false
}
```

Reply:
Sorry Jayson I wasn't clear ;( 
I mean we should check for rev == "" and error out if we don't have one vs later on in this call (l68)

Reply:
Thanks for the clarify, I see what you mean, the one that goes into this DAO **should** have the rev part.

I am in the process of revising this PR based on your comments, do you think it would be a good idea for us to keep the same behavior in helm history as helm chart(add the `v` command to view overridden values, and also the ability to toggle standard and overridden values via `v`)? 🤔️